css/src/atomics/table.scss

```diff
@@ -0,0 +1,108 @@
+/* stylelint-disable selector-no-qualifying-type */
+
+/* stylelint-disable selector-max-compound-selectors */
+
+/* stylelint-disable no-descending-specificity */
+
+.table {
+	width: 100%;
+	font-size: $font-size-8;
+	table-layout: auto;
+
+	caption {
+		padding: $font-size-6;
+		border-width: 0 0 $table-border-width;
+		border-style: solid;
+		border-color: $border;
+		font-size: $table-caption-size;
+		font-weight: $weight-semibold;
+		letter-spacing: $table-caption-spacing;
+		text-align: $user-right;
+		text-transform: uppercase;
+	}
+
+	td,
+	th {
+		display: table-cell;
+		word-wrap: break-word;
+		padding: $table-cell-padding-xxs $table-cell-padding-xs;
+		border-top: $table-border-width solid $border;
+		border-right: 0;
+		border-bottom: 0;
+		border-left: 0;
+		border-style: solid;
+		line-height: 1.5;
+		vertical-align: top;
+
+		&:not([align]) {
+			text-align: $user-left;
+		}
+
+		&.nowrap {
+			white-space: nowrap;
+		}
+
+		&.is-narrow {
+			width: $table-cell-narrow-width;
+		}
+	}
+
+	th {
+		&:not([scope='row']) {
+			border-top: 0;
+			border-bottom: $table-border-width;
+		}
+	}
+
+	&.table-striped {
+		tbody tr:nth-of-type(odd) {
+			background-color: $box-shadow-color-light;
+		}
+	}
+
+	&.min {
+		width: unset;
+	}
+
+	&.has-inner-borders {
```

Comment:
Let's hold on this until tomorrow's sync and thing about what we want to do with `has-*` modifiers and `is-*` modifiers.

Reply:
Oh, you're right, I was thinking about changing it but forgot 🤦🏻‍♀️ Let's sync tomorrow!